lib/metadata/BucketInfo.js

```diff
@@ -211,6 +263,27 @@ export default class BucketInfo {
         return this;
     }
     /**
+     * Get the website configuration information
+     * @return {object} websiteConfiguration
+     */
+    getWebsiteConfiguration() {
+        return this._websiteConfiguration;
+    }
+    /**
+     * Set website configuration information
+     * @param {object} websiteConfiguration - configuration for bucket website
+     * @return {BucketInfo} - bucket info instance
+     */
+    setWebsiteConfiguration(websiteConfiguration) {
```

Comment:
OK, I removed the check if it's an instanceof. As for the function itself, it's used to set the websiteConfiguration of the bucket before updating the bucket in MD.